src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,95 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import React, { useState } from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  const [activePage, setActivePage] = useState('1');
```

Comment:
The `activePage` state is initialized with a string '1', but it should be a number to maintain consistency with the type of `currentPage` which is also a number.